sbncode/CAFMaker/FillReco.cxx

```diff
@@ -10,6 +10,10 @@
 #include "FillReco.h"
 #include "RecoUtils/RecoUtils.h"
 
+#include "larevt/SpaceCharge/SpaceCharge.h"
+#include "larevt/SpaceChargeServices/SpaceChargeService.h"
+#include "larcore/CoreUtils/ServiceUtil.h"
```

Comment:
These will not be needed after the change below.
```suggestion
```

Reply:
Done! And moved `#include "larevt/SpaceCharge/SpaceCharge.h"` into `FillReco.h` from `FillReco.cxx`.